utilities/transactions/optimistic_transaction_impl.h

```diff
@@ -64,6 +64,8 @@ class OptimisticTransactionImpl : public TransactionBaseImpl {
   // Should only be called on writer thread.
   Status CheckTransactionForConflicts(DB* db);
 
+  void ClearBatch() override { Clear(); }
```

Comment:
Should we assert here
`assert(txn_state_.load() == COMMITED)`
We may even check it and return false if it does not make sense to clear the txn